### ISSUE TITLE
Log in in Portainer by OAuth error "Failure Unauthorized"

### ISSUE THREAD

**Issue by seniyakk** [goauthentik/authentik#11744]
**Describe the bug**
I set up Oauth for Portainer last year, and everything worked fine. But today, I could not log in to Portainer via OAuth. I got "Failure Unauthorized", " Unable to login via OAuth".
I followed the instructions https://docs.goauthentik.io/integrations/services/portainer/  and created a new  Provider and Application for Portainer. But get the same error.
Nothing has changed in the configuration of services. I have only updated Authentik.
For this moment I see the same error with Portainer, Audiobookshelf.

It seems like there are errors in callback. I see that I am succef

**To Reproduce**

1. Go to 'Portainer'
2. Click on 'Login with OAuth'
3. See error "Failure Unauthorized", " Unable to login via OAuth".

**Expected behavior**
Login in Portainer

**Screenshots**
<img width="1678" alt="Screenshot 2024-10-20 at 09 28 21" src="https://github.com/user-attachments/assets/49ea21c8-5dc5-44c8-b577-4beffb1a9ab7">
<img width="1710" alt="Screenshot 2024-11-23 at 10 41 45" src="https://github.com/user-attachments/assets/338c9c14-0c84-47e0-a3ba-e55e6774181d">

**Logs**

<img width="1710" alt="Screenshot 2024-11-23 at 10 29 32" src="https://github.com/user-attachments/assets/bfd8ae5d-6f7f-4fd0-af5b-e9e5343e710f">

**Version and Deployment (please complete the following information):**

-   authentik version: 2024.10.4
-   portainer version: 2.21.2
-   Deployment: [docker-compose, traefik]

**Comment by miccico** [comment]
Did you find a solution to this? I have found the same behavior today. Nothing changed and logins not working. One strange thing i find is that it not only happens in portainer but in nextcloud as well. What both have in common is that i see a failed request on my reverse proxy towards the authentik instance that translates to "method not allowed". In some internet research i saw people writing about this happening when the request is made as a GET but this error happens to me with POST as well. Nothing in the logs of authentik that i could find. i am on version 2024.10.0

[31/Oct/2024:23:11:06 +0100] xx.xxxxx.xxxxx.xx:8443 - 10.xx.xx.21 - 7EhE8wN2UU29AxxxxxxxxxxxxxxxxylfoDJBbgj "POST /application/o/token/ HTTP/2.0" 405 23 "-" "Go-http-client/2.0" "-"
[31/Oct/2024:23:11:06 +0100] xx.xxxxx.xxxxx.xx:8443 - 10.xx.xx.21 - - "POST /application/o/token/ HTTP/2.0" 405 23 "-" "Go-http-client/2.0" "-"

Thanks

Michael

**Comment by seniyakk** [comment]
No. I still have it. I have reviewed many forums but have not gotten a solution yet. Such a problem is not only with Portainer but other services as well.

**Comment by ThorTheStorm** [comment]
I just had this problem as well.
Followed the instructions for [Portainer at Authentik](https://docs.goauthentik.io/integrations/services/portainer/).

Adding the correct scopes from Authentik in the Portainer-Oauth configuration fixed this for me.

**Comment by start-the-fire** [comment]
I'm facing the same issue as well. Authorization just stopped working for portainer, wikiJS and nextcloud on my side as well
